fastlane/README.md

```diff
@@ -56,6 +56,11 @@ Upload a snapshot release
 fastlane android prepare_next_version
 ```
 Prepare next version
+### android deploy_amazon
+```
+fastlane android deploy_amazon
```

Comment:
this won't work since we deleted it right?

Reply:
oh right, just a README tho, I can delete it later